listenbrainz/webserver/templates/user/base.html

```diff
@@ -16,9 +15,6 @@
         <li {{ 'class=active' if active_section=='recommendations' }}><a href="{{ url_for('user.recommendation_playlists', user_name=user.musicbrainz_id) }}">Recommendations</a></li>
         <li {{ 'class=active' if active_section=='collaborations' }}><a href="{{ url_for('user.collaborations', user_name=user.musicbrainz_id) }}">Collaborations</a></li>
       </ul>
-
-    {% block profile_content %} {% endblock %}
-
```

Comment:
Why was this removed?
I'm not so well versed in this templating system, but since this is a base for other templates I believe this profile_content block can be extended and some content added to it like it is done here: https://github.com/metabrainz/listenbrainz-server/blob/master/listenbrainz/webserver/templates/recommendations_cf_recording/top_artist.html#L3-L5

Reply:
I actually decided to move this from under the listens to the `My profile` section.

Reply:
I don't think I understand your comment.
I just see `{% block profile_content %} {% endblock %}`  removed and not added anywhere else.